70_Pushover.pm

```diff
@@ -1330,6 +1361,8 @@ sub Pushover_SetMessage2 ($$$$) {
                   if ( defined( $hash->{READINGS}{$rAckBy} ) );
                 delete $hash->{READINGS}{$rCancelId}
                   if ( defined( $hash->{READINGS}{$rCancelId} ) );
+                delete $hash->{READINGS}{$rAckByDev}
+                  if ( defined( $hash->{READINGS}{$rAckByDev} ) );
```

Comment:
3) The actual device that acknowledge the message is being recorded now - this is especially useful when sending to groups